examples/music_drop_fade.py

```diff
@@ -170,7 +170,8 @@ def main():
     pg.scrap.init()
     pg.SCRAP_TEXT = pg.scrap.get_types()[0]  # TODO remove when scrap module is fixed
     clipped = pg.scrap.get(pg.SCRAP_TEXT).decode(
-        "UTF-8")  # store the current text from the clipboard TODO remove decode
+        "UTF-8"
+    )  # store the current text from the clipboard TODO remove decode
```

Comment:
minor code cleanliness nitpick, the comment can go on a new line, so that the line of code stays on the same line